javascript/08_stream_worker(javascript).ipynb

```diff
@@ -118,11 +166,11 @@
     "const jsc8 = require(\"jsc8\");\n",
     "\n",
     "// ----- simple way  -----\n",
-    "const client = new jsc8(global_url);\n",
+    "const client = new jsc8(fed_url);\n",
```

Comment:
Please also fix the "global_url" in the commented section below, on line 179 and 186